client/views/room/contextualBar/Call/Jitsi/CallJitsiWithData.js

```diff
@@ -155,41 +155,31 @@ const CallJitsiWithData = ({ rid }) => {
 	});
 
 	useEffect(() => {
-		let shouldDispose = false;
-
-		async function fetchData() {
-			if (!accepted || !jitsi) {
-				return;
-			}
-
-			const clear = () => {
-				jitsi.off('HEARTBEAT', testAndHandleTimeout);
-				jitsi.dispose();
-			};
+		if (!accepted || !jitsi) {
+			return;
+		}
 
-			try {
-				if (jitsi.needsStart) {
-					jitsi.start(ref.current);
-					await updateTimeout(rid, true);
-				} else {
-					await updateTimeout(rid, false);
-				}
-			} catch (error) {
-				dispatchToastMessage({ type: 'error', message: t(error.reason) });
-				clear();
-				handleClose();
+		const clear = () => {
+			jitsi.off('HEARTBEAT', testAndHandleTimeout);
+			jitsi.dispose();
+		};
+		const none = () => {};
```

Comment:
```suggestion
```
If we decide to change the return statement, then we could also erase this line.

Reply:
Done.